contracts/src/chainlink/cairo/ocr2/IAggregator.cairo

```diff
@@ -21,6 +21,7 @@ func NewTransmission(
     config_digest : felt,
     epoch_and_round : felt,
     reimbursement : felt,
+    gas_price : felt,
```

Comment:
nit: I'd put the `gas_price` next to `juels_per_fee_coin` as they are of the same type - input observations.

Also, as this event change our parsing of the event needs to be updated and the `TestNewTransmissionEvent_Parse` test in `ocr2/events_test.go`